src/styles/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin common-margins {
+  margin: 16px;
+}
+
+@mixin upper-margins {
+  margin-top: 16px;
+}
```

Comment:
Do not create mixins like that this looks like a variable with one property and also in this case you have the same value for both of them so better create a variable with value 16px